.github/ISSUE_TEMPLATE/release.md

```diff
@@ -14,7 +14,7 @@ Minor and major releases:
   - See current actions usage with: `rg --no-filename --hidden uses: | awk '{$1=$1;print}' | sort | uniq`
   - Use global search and replace to update them to the latest tags
 - [ ] Check for new or duplicate contributors since the last release:
-  `comm -13 <(git log LAST_TAG --format="%aN <%aE>" | sort | uniq) <(git log --format="%aN <%aE>" | sort | uniq)`.
+  `comm -13 (git log v4.3.0 --format="%aN <%aE>" | sort | uniq | psub) (git log --format="%aN <%aE>" | sort | uniq | psub)`.
```

Comment:
Just one question does bumpversion handle bumping to one version behind?

Reply:
I don't think so. But there is some shell magic one could do with `git tag` to get the most recent release.